src/ipred.rs

```diff
@@ -116,7 +116,7 @@ wrap_fn_ptr!(pub unsafe extern "C" fn cfl_pred(
     topleft: *const DynPixel,
     width: c_int,
     height: c_int,
-    ac: *const i16,
+    ac: &[i16; 1024],
```

Comment:
Could we write these `1024`s as `32 * 32` as they are in `int16_t ac[32 * 32];` in C?

Reply:
I'd rather introduce a constant rather than repeating the same expression everywhere, will add that to the PR before merging.